File: JavaScript/1020.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
let lines = input.split('\n');

let idadeDias = parseInt(lines.shift());

let anos = 0, meses = 0;

while( idadeDias >= 365 ) {
    idadeDias -= 365;
    anos++;
}

while( idadeDias >= 30 ) {
    idadeDias -= 30;
    meses++;
}

console.log(anos + " ano(s)");
console.log(meses + " mes(es)");
console.log(idadeDias + " dia(s)");

File: C/1020.c
#include <stdio.h>

int main() {
    int dias;

    scanf("%d", &dias);

    int a = 0, m = 0, d = 0;

    while( dias >= 365 ) {
        a++;
        dias -= 365;
    }

    while( dias >= 30 ) {
        m++;
        dias -= 30;
    }

    while( dias >= 1 ) {
        d++;
        dias--;
    }

    printf("%d ano(s)\n", a);
    printf("%d mes(es)\n", m);
    printf("%d dia(s)\n", d);

    return 0;
}   